### PR TITLE
update redis@ service template

### PR DESCRIPTION
**Pull request by ariva07**
# Description

Fixes listening on only loopback interface

## Issues Resolved

https://github.com/sous-chefs/redisio/issues/439

## Check List

- [x] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a CHANGELOG entry.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by Xorima** [comment]
Hey @ariva07 

Can you please add a changelog entry for this then we can get it merged and released? 

Thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 
 # redisio
 
+## 4.2.1 (2021-05-07)
```
Comment (ramereth) [inline CHANGELOG.md:4]:
```suggestion
## Unreleased
```

**Review by ramereth — CHANGES_REQUESTED** [review]
Please don't add the version in the changelog. We have some automation that will do that once we merge it.

File: CHANGELOG.md (hunk 1 above)
Comment (ariva07) [reply CHANGELOG.md]: Thanks, fixed

**Review by ramereth — APPROVED** [review]

**Comment by kitchen-porter** [comment]
Released as: [6.2.4](https://github.com/sous-chefs/redisio/releases/tag/6.2.4)

**Pull request merged**